docs/source/samples/SpatialDetection/spatial_calculator_multi_roi.rst

```diff
@@ -0,0 +1,42 @@
+Spatial Calculator Multi-ROI
+============================
+
+This example shows how one can use multiple ROIs with a single :ref:`SpatailLocationCalculator` node. A similar logic could be used as a simple depth line
```

Comment:
Recent documentation builds were failing due to a typo: Spat**ai**lLocationCalculator. Fixed it in https://github.com/luxonis/depthai-python/pull/336/commits/8df2a1015497c384d125bd461c7d79fa6b2cbd40